configuration/rules-dsl.md

```diff
@@ -442,7 +442,8 @@ val Number epoch = (MyDateTimeItem.state as DateTimeType).zonedDateTime.timeInMi
 val Number nowEpoch = now.millis
 
 // Convert DateTimeType to Joda DateTime
-val joda = new DateTime((MyDateTimeItem.state as DateTimeType).zonedDateTime.timeInMillis)
+val joda = new DateTime(MyDateTimeItem.state.toString)
+val joda = new DateTime((MyDateTimeItem.state as DateTimeType).zonedDateTime.toInstant.toEpochMilli)
```

Comment:
Maybe we should call this value different.
Technically it would be an error to initialize a variable two times with the same name and i think openHAB will probably reject the rules file, if it is done like this.

I know this is only an example, but sometimes people paste docs code to try something out and we should take care of code that works right after pasting it into a file.

My suggestion would be:
```
val jodaVariantOne = new DateTime(MyDateTimeItem.state.toString)
val jodaVariantTwo = new DateTime((MyDateTimeItem.state as DateTimeType).zonedDateTime.toInstant.toEpochMilli)
```
What do you think?

Reply:
`sometimes people paste docs code to try something out`
Yes, you're right. Me too.
Thank you!